wordpress-mobile/gutenberg-mobile
# Fix check for file urls

This PR updates the GB reference for one that fixes the test for local file URL

To test:
 - Start the main app WPiOS or WPAndroid app using this branch for RN
 - Add an image/video/media block
 - Select some media to upload
 - Go offline while the upload is going
 - Wait for fail
 - Go online again
 - Check if retry options show when tapping on the block.

PR submission checklist:
